src/Elastic.Apm/Report/Serialization/LabelsJsonConverter.cs

```diff
@@ -12,7 +12,7 @@ public override void WriteJson(JsonWriter writer, Dictionary<string, string> lab
 			foreach (var keyValue in labels)
 			{
 				writer.WritePropertyName(SerializationUtils.TrimToPropertyMaxLength(keyValue.Key));
-				writer.WriteValue(SerializationUtils.TrimToPropertyMaxLength(keyValue.Value));
+				writer.WriteValue(string.IsNullOrEmpty(keyValue.Value) ? "null" : SerializationUtils.TrimToPropertyMaxLength(keyValue.Value));
```

Comment:
JSON has `null` so I think we should use `Newtonsoft.Json.JsonWriter.WriteNull` if `keyValue.Value` is `null`.

BTW the only interesting case is when `keyValue.Value` is `null` - there's no reason to treat empty string any differently than any other non-null string so there is no need for `string.IsNullOrEmpty` in this case.

Reply:
Done